dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Google’s postal address library that powers Android and Chromium"
+DESCRIPTION="libaddressinput can provide information about \
+what input fields are required for a correct address input form for \
+any country in the world and can validate an address to highlight \
+input errors like missing required fields or invalid values."
+HOMEPAGE="https://github.com/googlei18n/libaddressinput"
+COPYRIGHT="2014 Google Inc."
```

Comment:
As google vendor rapidjson, they're subjected to it's [licenses and copyright holders][0]
```diff
-COPYRIGHT="2014 Google Inc."
+COPYRIGHT="2014 Google Inc.
+	2015 THL A29 Limited, a Tencent company
+	2015 Milo Yip
+	2006-2013 Alexander Chemeris
+	2002 JSON.org"
```

[0]: https://github.com/Tencent/rapidjson/blob/6043ad86887c3c8730ffe3b88ed2ade9991fa064/license.txt#L3